compiler/optimizer/Inliner.cpp

```diff
@@ -3087,6 +3087,11 @@ TR_HandleInjectedBasicBlock::createTemps(bool replaceAllReferences)
             if (tt->getNode()->getOpCode().isBranch() || tt->getNode()->getOpCode().isSwitch())
                tt = tt->getPrevTreeTop();
 
+            // If this treetop is an OSR point, a store cannot be placed between it and the 
+            // transition treetop in postExecutionOSR
```

Comment:
Do you need to check for postExecutionOSR here or is that done in some other way before this code gets called or when getOSRTransitionTreeTop is called ?